yagpdb-cc-website/docs/snippets/parseflags.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 11
+title: parseArgs
+---
+
+This snippet provides a reusable template which you can add to your custom commands.  
+It separates predefined flags from positional arguments within input; for example:
+- `-command -m 123 positional arg -f`
+    - might be parsed into:
```

Comment:
```suggestion
    ```
    -command -m 123 positional arg -f
    ```
might be parsed into:
```
weird bullets + indentation, code block is imo neater to look at